src/org/jetbrains/plugins/scala/compiler/ScalaHydraCompilerConfigurationPanel.scala

```diff
@@ -18,6 +19,8 @@ import scala.util.{Failure, Success}
   */
 class ScalaHydraCompilerConfigurationPanel(project: Project, settings: HydraCompilerSettings, hydraGlobalSettings: HydraApplicationSettings) extends HydraCompilerConfigurationPanel {
 
+  private val fileChooserDescriptor = new FileChooserDescriptor(false, true, false, false, false, false)
```

Comment:
Could you add the name of each passed argument (having `true` and `false` isn't very evocative :)). I'm just wondering if you could use named arguments or, if not possible, just add a `/*<argument>=*/ before each passed argument.